src/eu/packages/eu-component-icon/icon.test.js

```diff
@@ -0,0 +1,83 @@
+import path from 'path';
+import { renderTwigFile } from '@ecl-twig/test-utils';
+
+import brandedIcons from '@ecl/ec-resources-icons/dist/lists/branded.json';
```

Comment:
Now that we are fetching the eu package for the resources we should use the icons from that one, so here we would import from the EU package and not the EC one.

Reply:
Fixed in [b49cf0a
](https://github.com/ec-europa/ecl-twig/pull/15/commits/b49cf0a7c4788b648f51be03431c9254f3956ace) commit